src/gui/qgsmetadatawidget.cpp

```diff
@@ -522,8 +549,9 @@ void QgsMetadataWidget::setPropertiesFromLayer()
   mHistoryModel->setStringList( mMetadata.history() );
 }
 
-void QgsMetadataWidget::saveMetadata( QgsLayerMetadata &layerMetadata ) const
+bool QgsMetadataWidget::saveMetadata( QgsLayerMetadata &layerMetadata ) const
 {
+  bool bHasChanged = false;
```

Comment:
QGIS coding convention is not to use prefixes like "b", rename to hasChanged

Reply:
done